.github/ISSUE_TEMPLATE/release_checklist.md

```diff
@@ -16,6 +16,7 @@ assignees: ''
 - [ ] Wrap Y ()
 
 **Before release**:
+- [ ] Run `grep 'remove_version="vX.Y.Z"' **/*.py` to check if any deprecations and related tests should be removed in this version
```

Comment:
```suggestion
- [ ] Run `grep --include="*.py" -r 'remove_version="vX.Y.Z"' pygmt` from the base of the repository to check if any deprecations and related tests should be removed in this version
```
The command as written only works if inside the `pygmt` directory. An alternate solution is proposed that works from the base of the repository, or you could specify that the person needs to `cd pygmt` first.

Reply:
> The command as written only works if inside the `pygmt` directory.

For me, `grep 'remove_version="vX.Y.Z"' **/*.py` works in the base of the repository, but I'm using the macOS `grep` command. Doesn't it work for you?

Reply:
> > The command as written only works if inside the `pygmt` directory.
> 
> For me, `grep 'remove_version="vX.Y.Z"' **/*.py` works in the base of the repository, but I'm using the macOS `grep` command. Doesn't it work for you?

I had to do `grep 'remove_version="v0.7.0"' **/**/*.py` (an extra set of double asterisks) on Linux, but Meghan's `--include="*.py"` solution looks cleaner.

Reply:
I see. The command works for me because I'm using Zsh, which can expand `**` to nested directories.